mrt/meetings/participant.py

```diff
@@ -72,8 +72,13 @@ class MediaParticipants(PermissionRequiredMixin, MethodView):
     permission_required = ('manage_meeting', 'view_media_participant',
                            'manage_media_participant')
 
+    form_class = MediaParticipantEditForm
+
     def get(self):
-        return render_template('meetings/participant/media/list.html')
+        Form = custom_form_factory(self.form_class)
+        form = Form()
+        return render_template('meetings/participant/media/list.html',
+                                form=form)
```

Comment:
This line is over-indented. Pay attention to [indentation](https://www.python.org/dev/peps/pep-0008/#indentation).